gsad/src/gsad_http.c

```diff
@@ -465,38 +465,78 @@ char *
 gsad_message_new (const char *title, const char *msg,
                   cmd_response_data_t *response_data)
 {
-  gchar *xml, *resp;
-
-  xml = g_strdup_printf ("<gsad_response>"
-                         "<title>%s (GSA %s)</title>"
-                         "<message>%s</message>"
-                         "<token>%s</token>"
-                         "</gsad_response>",
-                         title,
-                         GSAD_VERSION,
-                         msg,
-                         "");
+  gchar *html;
+
+  html = g_strdup_printf (
+    "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\">"
+    "<html xmlns=\"http://www.w3.org/1999/xhtml\" lang=\"\">"
+    "<head>"
+      "<meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\"></meta>"
+      "<link rel=\"icon\" href=\"/img/favicon.gif\" type=\"image/gif\"></link>"
+      "<title>Greenbone Security Assistant</title>"
+      "<link rel=\"stylesheet\" type=\"text/css\" href=\"/css/gsa-base.css\"></link>"
+      "<link rel=\"stylesheet\" type=\"text/css\" href=\"/css/select2.min.css\"></link>"
+      "<link rel=\"stylesheet\" type=\"text/css\" href=\"/css/jquery-ui.structure.min.css\"></link>"
+      "<link rel=\"stylesheet\" type=\"text/css\" href=\"/css/jquery-ui.theme.min.css\"></link>"
+      "<link rel=\"stylesheet\" type=\"text/css\" href=\"/css/gsa-style.css\"></link>"
+      "<script src=\"/js/jquery-2.1.4.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/jquery-ui.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/select2.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/i18next-2.3.4.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/i18next-xhr-0.5.3.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/i18next-languagedetector-0.2.2.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/moment.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/moment-timezone-with-data.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/greenbone-ui.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/greenbone.js\" type=\"text/javascript\"></script>"
+      "<script src=\"/js/gsa_polyfill.js\" type=\"text/javascript\"></script>"
+    "</head>"
+    "<body>"
+    "<div class=\"gsa-main\"><div>"
```

Comment:
I really don't know if the css classes are still available. Maybe we just reduce this page body to something like
```html
<h1>Not found</h1>
<p>The requested content couldn't be found.</p>
```
without any css formatting.